### PR TITLE
Handle read of EPSG code along with UTM zone in ply files

### PR DESCRIPTION
**Pull request by lofrad**
Enables reading .ply files with the projection comment expressed as an EPSG code and injecting the relevant Proj object in the DSM metadata.

### CODE REVIEW

File: plyflatten/utils.py [hunk 1]
```diff
@@ -9,25 +9,39 @@ class InvalidPlyCommentsError(Exception):
...
+def crs_proj(utm_zone, crs_type="UTM"):
     """
     Return a pyproj.Proj object that corresponds
-    to the given utm_zone string
+    to the given utm_zone string or EPSG code
 
     Args:
-        utm_zone (str): UTM zone number + hemisphere (eg: '30N')
+        crs_type (str): 'UTM' (default) or 'EPSG'
+        utm_zone (str): UTM zone number + hemisphere (eg: '30N') or EPSG code
```
Comment (carlodef) [inline plyflatten/utils.py]: The ordering of these two docstring lines is reverted with respect to the ordering of the function arguments

File: plyflatten/utils.py [hunk 2]
```diff
@@ -9,25 +9,39 @@ class InvalidPlyCommentsError(Exception):
...
-        proj="utm", zone=zone_number, ellps="WGS84", datum="WGS84", south=(hemisphere == "S")
-    )
+    if crs_type == "UTM":
+        zone_number = utm_zone[:-1]
+        hemisphere = utm_zone[-1]
+        return pyproj.Proj(
+            proj="utm", zone=zone_number, ellps="WGS84", datum="WGS84", south=(hemisphere == "S")
+        )
+    elif crs_type == "EPSG":
+        return pyproj.Proj("epsg:{}".format(utm_zone))
```
Comment (carlodef) [inline plyflatten/utils.py]: the variable name `utm_zone` is a bit misleading as it actually contains either a UTM zone or a EPSG code. Shouldn't we rename it to something more generic, like `crs` for example?

File: plyflatten/utils.py [hunk 3]
```diff
@@ -36,12 +50,18 @@ def utm_zone_from_ply(ply_path):
         if s:
```
Comment (carlodef) [inline plyflatten/utils.py]: this block of code is now duplicated (i.e. in `epsg_code_from_comments` and in `crs_from_ply`). Could we deduplicate it by making `epsg_code_from_comments` more generic? I mean the code block from lines 46 to 51 of the `epsg-code` version of the file

**Review by carlodef — APPROVED** [review]
Thank you @lofrad for this pull request! It looks good to me, I only have three minor comments / suggestions below.

**Comment by lofrad** [comment]
Thank you @carlodef for your review. I have addressed the 3 points you mentioned in the commit 7e0380ee1abd0dd1cf646a36de0238de52d5d16c

**Pull request merged**